.env

```diff
@@ -1,27 +1,27 @@
 # used for testing in /tests/run-tests.sh
 export QONTRACT_SERVER_NAME=qontract-server
 export QONTRACT_SERVER_IMAGE=quay.io/app-sre/qontract-server
-export QONTRACT_SERVER_IMAGE_TAG=dc34527
+export QONTRACT_SERVER_IMAGE_TAG=ed1f3d5
 
-# used for testing in /tests/run-tests.sh
+# used for testing in /tests/run-tests-compose.sh
 export KEYCLOAK_NAME=keycloak
 export KEYCLOAK_IMAGE=quay.io/keycloak/keycloak
-export KEYCLOAK_IMAGE_TAG=17.0
+export KEYCLOAK_IMAGE_TAG=latest
 export KEYCLOAK_USER=admin
 export KEYCLOAK_PASSWORD=admin
 
-# used for testing in /tests/run-tests.sh
+# used for testing in /tests/run-tests-compose.sh
 export KEYCLOAK_CLI_NAME=keycloak_cli
 export KEYCLOAK_CLI_IMAGE=quay.io/app-sre/keycloak-config-cli
 export KEYCLOAK_CLI_IMAGE_TAG=5.0.0-17.0.1
 
-# used for testing in /tests/run-tests.sh
+# used for testing in /tests/run-tests-compose.sh
 export VAULT_NAME=primary-vault
 export VAULT_NAME_SECONDARY=secondary-vault
 export VAULT_IMAGE=quay.io/app-sre/vault
-export VAULT_IMAGE_TAG=1.9.10
-export VAULT_SECONDARY_IMAGE_TAG=1.12.4
-export VAULT_ADDR=http://127.0.0.1:8200
+export VAULT_IMAGE_TAG=1.17.1
+export VAULT_SECONDARY_IMAGE_TAG=1.17.1
+export VAULT_ADDR=http://primary-vault:8200
```

Comment:
Is there a need for a secondary vault addr? Is the VAULT_ADDR variable overloaded to represent both instances?

Reply:
Using an env variable for each instance allows vault commands to be passed in using `-address=` to rely less on ensuring the variables are set in subshells during testing.  The `VAULT_SECONDARY_IMAGE_TAG` is redundant since both instances will be on the same version.

Reply:
I still don't follow a missing "VAULT_PRIMARY_ADDR" and "VAULT_SECONDARY_ADDR". Does this question make sense?

Reply:
Hopefully this is what you mean: https://github.com/app-sre/vault-manager/pull/135/commits/2881f84201564b02928b2454fc8c4ba0623e3633

Apologies for the oversight.